staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered_test.go

```diff
@@ -173,7 +173,7 @@ func TestBufferedBackendShutdownWaitsForDelegatedCalls(t *testing.T) {
 
 	// Run backend, process events, wait for them to be batched and for delegated call to start.
 	stopCh := make(chan struct{})
-	backend.Run(stopCh)
+	go backend.Run(stopCh)
```

Comment:
Why? is that necessary?

Reply:
I thought that the [issue](https://github.com/kubernetes/kubernetes/issues/7923) required changing the pattern where an object was instantiated and a method defined on that object was run to be changed to a goroutine.

Reply:
it says 

> We should pick one and switch to it all over.

and the previous PR referenced on this issue changed the Run methods to be launched as goroutines.